kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1951,6 +1951,16 @@ object KyuubiConf {
       .stringConf
       .createWithDefault("1")
 
+  val BATCH_INFO_INTERNAL_REDIRECT = buildConf("kyuubi.batch.info.internal.redirect")
+    .serverOnly
+    .doc("When set to true, the batch info is retrieved by forwarding the request to the " +
+      "corresponding Kyuubi instance using the internal REST client. If false, the batch info is " +
```

Comment:
let's make the "corresponding" clear

to the corresponding Kyuubi instance using the internal REST client.
=>
to the Kyuubi instance that submitted this job via internal RPC.

Reply:
I remember there may have been some state inconsistent risk for batch v1 implementation previously, is that still true?

Reply:
I can't think of any risks with the current V1 implementation